apps/search/js/providers/suggestions.js

```diff
@@ -71,17 +53,14 @@
       ul.setAttribute('role', 'listbox');
 
       items.forEach(function each(item) {
```

Comment:
Should there be an upper limit on the number of suggestions? The number of suggestions provided by Google seems quite high for example.